src/test/java/softvisionProject/POM/CareersPage.java

```diff
@@ -82,9 +102,9 @@ public void clickLocation(String selectLocation) {
     }
 
     public void clickExpertise(String selectExpertise) {
```

Comment:
What is the diff between clickExpertise and selectExpertise ?

Reply:
clickExpertise is the old method for dealing with drop down lists, similar to clickLocations which is now being replaced by selectLocations

Reply:
if is old remove it